core/src/migrations/sqlite/1666713816863-FTS.ts

```diff
@@ -4,9 +4,13 @@ export class FTS1666713816863 implements MigrationInterface {
     name = 'FTS1666713816863';
 
     public async up(queryRunner: QueryRunner): Promise<void> {
-        await queryRunner.query(
-            `CREATE VIRTUAL TABLE bible_search USING fts5(verse, versionUid UNINDEXED, versionBook UNINDEXED, versionChapter UNINDEXED, versionVerse UNINDEXED);`
-        );
+        try {
+            await queryRunner.query(
+                `CREATE VIRTUAL TABLE bible_search USING fts5(verse, versionUid UNINDEXED, versionBook UNINDEXED, versionChapter UNINDEXED, versionVerse UNINDEXED);`
+            );
+        } catch (error) {
+            console.error('failed to create virtual table bible_search', error)
```

Comment:
Ok, i think that works. Could you add a little context to the console error, e.g. "(This is an expected error if you use SQLite versions without fts support. It can be safely ignored as long as you don't enable/use full text search features in BibleEngine)".

I remembered that i had to detect the db-driver in some other case and looked it up quickly. I think you can access the driver type via `queryRunner.connection.driver.options.type`. One could argue both ways if this is cleaner or the other more solid. I'm ok with either. However a code comment explaining the situation would be nice in any case.

Reply:
Thanks Chris, great feedback, hope you had a good Christmas and new years! 

Updated the console message.

I thought about trying to be smart about detecting the driver type, but it gets really tricky because the same driver might have different fts5 support, or expo could be updated to have fts5 support etc... so didn't want to go down that rabbit hole!